lib/modules/manager/poetry/extract.ts

```diff
@@ -37,7 +38,12 @@ export async function extractPackageFile(
       if (dep.currentValue) {
         pythonVersion = dep.currentValue;
       }
-      return null;
+      return {
+        ...dep,
+        datasource: DockerDatasource.id,
```

Comment:
we should use our containerbase python as source. there are a lot docker tags which can cause issues with poetry versioning.

Reply:
Is that already available for renovate or it needs to be introduced?

Reply:
it's available.

https://github.com/containerbase/python-prebuild

Reply:
Raised https://github.com/renovatebot/renovate/pull/24463